### DISCUSSION TITLE
Convert to PPT

### DISCUSSION THREAD

**Discussion by inetmarketeers** [OfficeDev/Open-XML-SDK#958] [Q&A]
Is there anything in the library to convert to PPT from Doc or another format?

**Comment by ThomasBarnekow** [comment]
There is definitely nothing in the SDK, which, by design, only offers basic and direct access to the Open XML markup. What you are looking for would be a very involved transformation feature.

**Comment by inetmarketeers** [comment]
There are a few commercial API that do a nice job, but trying to find alternatives. Certainly we prefer not to burden users with subscription type services to do this.

Thanks for checking.

Chris

**Reply by ThomasBarnekow** [reply]
@twsouthwick, you can close this issue.
